lib/browser/browser.js

```diff
@@ -37,13 +37,19 @@ function OktaAuthBuilder(args) {
     issuer: util.removeTrailingSlash(args.issuer),
     authorizeUrl: util.removeTrailingSlash(args.authorizeUrl),
     userinfoUrl: util.removeTrailingSlash(args.userinfoUrl),
+    tokenUrl: util.removeTrailingSlash(args.tokenUrl),
+    grantType: args.grantType,
     redirectUri: args.redirectUri,
     httpRequestClient: args.httpRequestClient,
     storageUtil: args.storageUtil,
     transformErrorXHR: args.transformErrorXHR,
     headers: args.headers
   };
 
+  if (this.options.grantType === 'authorization_code' && !sdk.features.isPKCESupported()) {
```

Comment:
Don't we also need this statement whenever `token.getWith*` methods are called? Developers should be able to init their authClient with some config but override it when calling `getWith*`.

Reply:
A developer CAN override `grantType` in these calls, but we are not checking for PKCE support. I did have this logic + tests, but after talking with @robertdamphousse-okta we decided to leave it out until requested, as we want to encourage setting `grantType` up front so that token auto-renew works properly.

Reply:
It would be trivial to bring it back: https://github.com/okta/okta-auth-js/pull/205/commits/dc1adca492f7d7cfcb3bba523d9c1d13b3164e5d#diff-2c8402d8ca2c5c319eb971f5bdf5ac4dR37

Reply:
adding this back in.